test/unit/math/prim/fun/lb_transform_test.cpp

```diff
@@ -5,9 +5,6 @@
 
 TEST(prob_transform, lb) {
   EXPECT_FLOAT_EQ(exp(-1.0) + 2.0, stan::math::lb_constrain(-1.0, 2.0));
-  EXPECT_THROW(
-      stan::math::lb_constrain(7.9, -std::numeric_limits<double>::infinity()),
-      std::domain_error);
```

Comment:
I think go back in history with these checks: https://github.com/stan-dev/math/blob/89c8c8c130f40ccd1b0fb42bdbd2049644939461/test/unit/math/prim/fun/lb_transform_test.cpp

Reply:
We need prim checks that `lb_constrain(vector, real)` and `lb_constrain(vector, vector)` are behaving well with infs (not transforming when there is an inf, transforming otherwise)

Reply:
Oh yeah that's a good point